.github/workflows/ci-test.yaml

```diff
@@ -57,10 +57,17 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
+      - name: Normalize YAML
+        run: |
+          VERSION=v4.30.5 BINARY=yq_linux_amd64
+          wget https://github.com/mikefarah/yq/releases/download/${VERSION}/${BINARY} -O ./yq
+          chmod +x ./yq
+          # OpenAPI parser fails to parse YAML templates (*, <<:). Expand them out here.
```

Comment:
Oh nice. I totally didn't know about this fix/solution when I un-templated.

Reply:
It ain't pretty. But the bug is in the linter, not in our spec.